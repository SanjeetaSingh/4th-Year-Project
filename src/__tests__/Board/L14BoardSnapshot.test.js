import renderer from 'react-test-renderer'

import Level14 from '../../Level14/L14-Board/L14-Board'

it('snapshot test level 14', () => {
    const tree =  renderer.create(<Level14 />).toJSON()
    expect(tree).toMatchSnapshot();
});