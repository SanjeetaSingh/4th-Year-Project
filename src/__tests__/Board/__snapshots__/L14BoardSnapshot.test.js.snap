// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test level 14 1`] = `
<div
  className="container"
>
  <div
    className="boardcontainer2"
  >
    <div
      className="board14"
    >
      <div
        className="box"
        data-col={0}
        data-row={0}
      />
      <div
        className="box"
        data-col={1}
        data-row={0}
      />
      <div
        className="box"
        data-col={2}
        data-row={0}
      >
        <img
          alt="f"
          class="cat"
          id="catOne"
          src="assets/cat.png"
        />
      </div>
      <div
        className="box"
        data-col={3}
        data-row={0}
      />
      <div
        className="box"
        data-col={0}
        data-row={1}
      />
      <div
        className="box"
        data-col={1}
        data-row={1}
      >
        <img
          alt="d"
          class="image"
          id="dog"
          src="assets/dog.png"
        />
      </div>
      <div
        className="box"
        data-col={2}
        data-row={1}
      />
      <div
        className="box"
        data-col={3}
        data-row={1}
      />
      <div
        className="box"
        data-col={0}
        data-row={2}
      >
        <img
          alt="f"
          class="cat"
          id="catTwo"
          src="assets/cat.png"
        />
      </div>
      <div
        className="box"
        data-col={1}
        data-row={2}
      />
      <div
        className="box"
        data-col={2}
        data-row={2}
      />
      <div
        className="box"
        data-col={3}
        data-row={2}
      />
      <div
        className="box"
        data-col={0}
        data-row={3}
      />
      <div
        className="box"
        data-col={1}
        data-row={3}
      />
      <div
        className="box"
        data-col={2}
        data-row={3}
      />
      <div
        className="box"
        data-col={3}
        data-row={3}
      >
        <img
          alt="f"
          class="bone"
          id="food"
          src="assets/food.png"
        />
      </div>
    </div>
  </div>
  <div
    className="bottom-panel"
  >
    <div
      class="border"
    >
      <div
        class="level13Contain"
      >
        <h2>
          Level 14:
        </h2>
        <div
          class="speech13"
        >
          Varibles again! The variable in this level is setting a boolean to either true or false. In the input field type either true or false and this boolean will be used in the if statement to check if the cat is present or not and when to bark. Use the same amount or less amount of commands mentioned in the top right corner.
          <br />
           Good luck! 
          <br />
           (To learn more about while, understand the while loop condition, how variables work and about if statements click the i below)
        </div>
        <div
          class="borderPanel"
        >
          <div
            class="toppart13"
          >
            <div
              class="containing"
            >
              <p
                class="number"
                data-testid="counter"
                id="count"
              >
                0/5
              </p>
            </div>
            <p
              class="titles"
            >
              Enter Sequence:
            </p>
            <br />
            <div
              class="buttons-wrapper12"
            >
              <button
                class="seemingly-inner-button12"
                disabled={false}
                onClick={[Function]}
              >
                <i
                  class="fa fa-info"
                />
                    Variables
              </button>
              <p
                class="subtitles"
              >
                Variable Declaration
              </p>
            </div>
            <form
              class="var14"
            >
              boolean catPresent =  
              <input
                class="input"
                id="var1"
                onChange={[Function]}
                size="5"
              />
                ;       /**Hint: We want to check if the cat is true**/
              <br />
            </form>
            <p
              class="lists"
              id="action"
            />
          </div>
        </div>
        <div
          class="outside"
        >
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Left
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Up
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Down
          </button>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Right
          </button>
          <div
            class="buttons-wrapper5"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
              type="submit"
            >
              While
            </button>
          </div>
          <div
            class="buttons-wrapper5"
          >
            <button
              class="seemingly-inner-button"
              disabled={false}
              onClick={[Function]}
            >
              <i
                class="fa fa-info"
              />
            </button>
            <button
              class="button"
              disabled={false}
              onClick={[Function]}
            >
              If Statement
            </button>
          </div>
          <button
            class="button"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Bark
          </button>
          <button
            class="buttonClear"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
            Clear
          </button>
          <button
            class="buttonSub"
            disabled={false}
            onClick={[Function]}
            type="submit"
          >
             Submit
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
